src/Server.js

```diff
@@ -85,11 +85,15 @@ class Server {
     const Config = await IoC.use("Config");
 
     this.app.get("/", (req, res) => {
-      res.json({
+      const defaultAxeResponse = {
         name: "AXE API",
         description: "The best API creation tool in the world.",
         aim: "To kill them all!",
-      });
+        docs: `http://localhost:${Config.Application.port}/docs`,
+        routes: `http://localhost:${Config.Application.port}/docs/routes`,
+        "learn-more": "https://axe-api.github.io",
+      };
+      res.json(Object.keys(Config.Application).includes('defaultResponse') ? Config.Application.defaultResponse : defaultAxeResponse);
```

Comment:
We don't have to check `defaultResponse` value if it is defined or not. Also, we should create another PR on the [axe-api-template](https://github.com/axe-api/axe-api-template/blob/master/app/Config/Application.js) for default response. So that if a new user creates a new Axe API project, defaultResponse object will be there.